src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <>
+    <div className="tabs is-boxed">
+      <ul>
+        { tabs.map(({ id, title, content }) => (
+          <li
+            key={id}
+            className={id === selectedTab.id
+              ? 'is-active'
+              : null
+            }
+            data-cy="Tab"
+          >
+            <a
+              href={`#${id}`}
+              data-cy="TabLink"
+              onClick={() => id !== selectedTab.id
+                && onTabSelected({ id, title, content })}
+            >
+              {`${title}`}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div
+      className="block"
+      data-cy="TabContent"
+    >
+      {selectedTab.content}
+    </div>
```

Comment:
better to move it to the App.tsx

Reply:
was recommended to move from App.jsx to Tabs.jsx in order to pass tests [link](https://mate.academy/chats/8412?selectedMessageId=1425221)